src/app/routes/regex/index.js

```diff
@@ -27,3 +28,6 @@ export const mediaRadioAndTvRegexPathsArray = buildMediaRoutes(
 );
 
 export const mediaDataRegexPath = buildMediaDataRoutes(servicesWithRadioOrTv);
+
+export const mediaAssetPageRegexPath = `/:service(${serviceRegex})/:assetUri(${assetUriRegex}):amp(${ampRegex})?`;
+export const mediaAssetPageDataRegexPath = `${mediaAssetPageRegexPath}.json`;
```

Comment:
Probably a non-issue, but this will also match invalid URLs due to `mediaAssetPageRegexPath` including `.amp`

Reply:
@ryanmccombe Good spot, I tried it `/pidgin/tori-49450859.amp.json` it doesn't pass the route validation